app/components/assets/form.tsx

```diff
@@ -420,10 +420,20 @@ export const AssetForm = ({
   );
 };
 
-const Actions = ({ disabled }: { disabled: boolean }) => (
+const Actions = ({
+  assetId,
+  disabled,
+}: {
+  assetId?: string;
```

Comment:
If we decide to keep it like this, it shoudl not be optional. It should be required and it can be string or null. That way we make sure the DX is better and the developer is always reminded when using <Actions> they have to pass the id. Like this we have a case where they might have the ID available but not pass it as its not required by <Actions>'s props.